packages/edit-post/src/components/layout/style.scss

```diff
@@ -68,11 +68,66 @@
 }
 
 .edit-post-layout__content {
-	position: relative;
 	display: flex;
-	min-height: 100%;
 	flex-direction: column;
 
+	// These rules are specific to mobile and small breakpoints.
+	min-height: 100%;
+	position: relative;
+
+	// We scroll the main editing canvas, the sidebar, and the block library
+	// separately to prevent scroll bleed. Because the navigation sidebar menu
+	// has "flyout" menus, we can't scroll that independently. Once we can
+	// scroll the navigation sidebar menus independantly: we should simplify
+	// these rules.
+	// In the meantime: if a user has a small screen and lots of menu items
+	// (added by plugins, for instance) in the navigation menu,
+	// they have to be able to scroll. To accommodate the flyout menus,
+	// we scroll the `body` element.
+	@include break-medium() {
+		// Because the body element scrolls the navigation sidebar, we
+		// have to use position fixed here.
+		// Otherwise you would scroll the editing canvas out of view
+		// when you scroll the sidebar.
+		position: fixed;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		top: $header-height + $admin-bar-height; // Because this is scoped to break-medium and larger, the admin-bar is always this height.
```

Comment:
These comments after each attribute get really long and make for quite long line-lengths. If they could be moved around that'd be grand, but having them relate to the attributes is handy too, so no worries if you wanna leave 'em as-is.